.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
### Your Summary

![Screenshot from 2022-09-20 18-04-59](https://user-images.githubusercontent.com/93039359/191266061-aad3ed11-6b0a-4241-b4a4-d7845bb38a5d.png)

### Descriptive Summary Example :heavy_check_mark:
![summary-descriptive](https://user-images.githubusercontent.com/93039359/191243176-550fc938-97e4-4cf7-969c-ce112fd46536.png)

- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: